src/multi_physics/CMakeLists.txt

```
@@ -1,13 +1,33 @@
-cmake_minimum_required (VERSION 3.5)
+cmake_minimum_required(VERSION 3.12)
 
-set(CMAKE_CXX_EXTENSIONS OFF)
+project(
+  pxrmp_qed
+  DESCRIPTION
+    "A header only c++ library (part of the PICSAR library) providing QED modules for Particle-In-Cell codes."
+  HOMEPAGE_URL "https://github.com/ECP-WarpX/picsar"
+  LANGUAGES CXX)
+  
+add_library(${PROJECT_NAME} INTERFACE)
```

Adding this alias will make `add_subdirectory` and `find_package` work with the same name:
```suggestion
add_library(${PROJECT_NAME} INTERFACE)
add_library(${PROJECT_NAME}::${PROJECT_NAME} ALIAS ${PROJECT_NAME})
```